ibridges/__main__.py

```diff
@@ -113,9 +114,34 @@ def _set_ienv_path(ienv_path: Union[None, str, Path]):
             return None
```

Comment:
Why do we need this part here? If the .ibridges folder does not exist or the ibridges_cli.json does not exist, we can simply create the folder and return none. That should be be independent from whether we have an ioenv_path.

Reply:
If users are using the default path and only one instance, they won't need an .ibridges folder, so to keep things clean we don't create one. I think there might be a reasonable number of people using ibridges this way and we don't want to clutter their home folder then.